app/screens/edit_profile/edit_profile.tsx

```diff
@@ -342,6 +383,33 @@ const EditProfile = ({
         returnKeyType: 'next',
     };
 
+    const renderProfilePicture = useCallback(() => {
+        if (lockedPicture) {
+            return (
+                <View style={styles.top}>
+                    <ProfilePicture
+                        author={currentUser}
+                        size={153}
+                        showStatus={false}
+                    />
+                </View>
+            );
+        }
+
+        if (!currentUser.isBot) {
```

Comment:
do we need to check for `isBot`? this seems weird to me

Reply:
absolutely.  removing it.